packages/toolbars-new/web/src/Toolbar/buttonsListCreator.tsx

```diff
@@ -204,17 +213,36 @@ const handleTitleButton = (buttonsList, index, editorCommands: editorCommands) =
 const handleButtonOnSave = (buttonsList, index, editorCommands: editorCommands) => {
   if (buttonsFullData[buttonsList[index].name].onSave) {
     const buttonName = buttonsList[index].name;
-
     if (Object.keys(textBlockButtons).includes(buttonName)) {
-      buttonsList[index].onSave = type => editorCommands.setBlockType(type);
+      buttonsList[index].onSave = type => {
+        editorCommands.setBlockType(type);
+      };
     } else if (buttonName === 'Alignment') {
       buttonsList[index].onSave = type => editorCommands.setTextAlignment(type);
     } else if (Object.keys(decorationButtons).includes(buttonName)) {
       buttonsList[index].onSave = type => {
-        if (buttonName === 'LINE_SPACING') {
+        if (['LINE_SPACING', 'FONTSIZE'].includes(buttonName)) {
           if (type) {
-            updateSpacing(type, editorCommands, buttonName);
-            setTimeout(() => editorCommands.loadSelectionState());
+            if (buttonName === 'LINE_SPACING') {
+              editorCommands.insertDecoration(decorationButtons[buttonName], {
+                dynamicStyles: type,
+              });
+              setTimeout(() => editorCommands.loadSelectionState());
+            } else if (buttonName === 'FONTSIZE') {
+              const data = { fontSize: type < 10 ? 10 : type > 96 ? 96 : type };
```

Comment:
another way to do it Math.min(96, Math.max(val, 10))
```suggestion
const fontSize = type < 10 ? 10 : type > 96 ? 96 : type
              const data = { fontSize };
```